packages/unified/src/unified.ts

```diff
@@ -0,0 +1,108 @@
+import { AmplitudeBrowser } from '@amplitude/analytics-browser';
+import {
+  AmplitudeSessionReplay,
+  SessionReplayOptions,
+  sessionReplayPlugin,
+  SessionReplayPlugin,
+} from '@amplitude/plugin-session-replay-browser';
+import {
+  IExperimentClient,
+  ExperimentConfig,
+  ExperimentPlugin,
+  experimentPlugin,
+} from '@amplitude/plugin-experiment-browser';
+import { BrowserClient, BrowserOptions, LogLevel } from '@amplitude/analytics-core';
+
+export interface UnifiedSharedOptions {
+  serverZone?: 'US' | 'EU';
+  instanceName?: string;
+  /**
+   * If logLevel is debug:
+   * - analytics.config.logLevel = debug
+   * - sr.config.logLevel = debug
+   * - sr.config.debugMode = true
+   * - experiment.config.debug = true
+   */
+  logLevel: LogLevel;
+}
+
+export type UnifiedOptions = UnifiedSharedOptions & {
+  analytics?: BrowserOptions;
+  sr?: Omit<SessionReplayOptions, keyof UnifiedSharedOptions>;
+  experiment?: Omit<ExperimentConfig, keyof UnifiedSharedOptions>;
+};
+
+export interface UnifiedClient extends BrowserClient {
+  initAll(apiKey: string, unifiedOptions?: UnifiedOptions): Promise<void>;
```

Comment:
Technically, we can use [function overloads](https://www.typescriptlang.org/docs/handbook/2/functions.html#function-overloads) to keep the same name `init()`. For example
```ts
  init(apiKey: string, unifiedOptions?: UnifiedOptions): Promise<void>;
  init(apiKey?: string, userIdOrOptions?: string | BrowserOptions, maybeOptions?: BrowserOptions): ReturnType<AmplitudeBrowser['init']>;
  init(
    apiKey?: string,
    userIdOrOptions?: string | BrowserOptions,
    maybeOptions?: BrowserOptions,
  ): Promise<void> | ReturnType<AmplitudeBrowser['init']>;
```

But i'd like to keep this as `initAll()` so that the implementation is simpler for now. Please let me know if you have a strong opinion to not use `initAll()` here.